src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHet.java

```diff
@@ -58,8 +58,9 @@
         final double pval = exactTest(hetCount, refCount, homCount);
 
         //If the actual phredPval would be infinity we will probably still filter out just a very large number
+        //Since the method does not guarantee precision for any p-value smaller than 1e-16, we can return the phred scaled version
         if (Math.abs(pval - 0.0) < 10e-60) {
-            return Pair.of(Integer.MAX_VALUE, Double.POSITIVE_INFINITY);
+            return Pair.of(sampleCount, -10.0 * Math.log10(MIN_NEEDED_VALUE));
```

Comment:
can you extract `-10.0 * Math.log10(MIN_NEEDED_VALUE)` as a constant? I dont want to keep recomputing it.